ostree-upload-server.py

```diff
@@ -76,10 +85,27 @@ def get_queue(self):
     def join(self, timeout=None):
         return self.queue.join(timeout)
 
+class Counter:
+    def __init__(self):
+        self.count = 0
+
+    def __enter__(self):
+        self.count += 1
+        print("counter now " + str(self.count))
+        return self.count
+
+    def __exit__(self, type, value, traceback):
+        self.count -= 1
+        print("counter now " + str(self.count))
 
 app = Flask(__name__)
-app.config["UPLOAD_FOLDER"] = tempfile.mkdtemp("upload")
+tempdir = tempfile.mkdtemp("upload")
+atexit.register(os.rmdir, tempdir)
+app.config["UPLOAD_FOLDER"] = tempdir
 
+latest_task_complete = time()
+latest_maintenance_complete = time()
+active_upload_counter = Counter()
```

Comment:
This should be fields on a class rather than module-level vars to ensure insular components.